docs/getting-started/v2-with-cash-app-pay.md

```diff
@@ -1,11 +1,11 @@
 ---
 layout: default
-title: Cash App Pay
+title: Checkout V2 with Cash App Pay
```

Comment:
On iOS this is just Cash App Pay

Reply:
I want to distinguish v2 and v3 each with cash app pay